test/cmd/policy.sh

```diff
@@ -125,6 +125,9 @@ os::cmd::expect_success_and_text 'oc adm policy add-cluster-role-to-group cluste
 os::cmd::expect_success_and_text 'oc adm policy add-cluster-role-to-user cluster-reader namespaced-user --dry-run' 'cluster role "cluster\-reader" added: "namespaced\-user" \(dry run\)'
 
 os::cmd::expect_success 'oc adm policy add-role-to-group view testgroup'
+os::cmd::expect_success 'oc adm policy add-cluster-role-to-group cluster-reader testgroup'
+os::cmd::expect_success 'oc adm policy add-cluster-role-to-user cluster-reader namespaced-user'
+
```

Comment:
Now that the following remove-cluster-role tests succeed because the target is present, add failure tests for remove-cluster-role on a missing target